### PR TITLE
ci: cancel an already running jobs

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [x] Other... Please describe: CI

## What is the current behavior?

if you push to branch several times, then on the CI unnecessary pipeline starts for each commit

## What is the new behavior?

Cancel an already running jobs 

https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idconcurrency

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -64,3 +64,7 @@ jobs:
                   npm publish dist/addon-tablebars
                   npm publish dist/icons
                   npm publish dist/taiga-schematics
+
+concurrency:
+    group: releaase-${{ github.head_ref }}
```
Comment (vladimirpotekhin) [inline .github/workflows/release.yml:69]: too much a )

File: .github/workflows/demo-integrations.yml [hunk 2]
```diff
@@ -40,3 +40,7 @@ jobs:
                   name: cypress-screenshots
                   path: projects/demo-integrations/cypress/snapshots
                   retention-days: 7
+
+concurrency:
+    group: demo-integration-${{ github.head_ref }}
```
Comment (MarsiBarsi) [inline .github/workflows/demo-integrations.yml:45]: integrations?

Comment (splincode) [reply .github/workflows/demo-integrations.yml]: fixed)

File: .github/workflows/release.yml (hunk 1 above)
Comment (splincode) [reply .github/workflows/release.yml]: fixed)

**Review by vladimirpotekhin — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

**Pull request merged**